### PR TITLE
fix(curriculum): typo error on css file 

### LINKED ISSUE #56284: Missing semicolon in a CSS rule.
### Describe the Issue

.price {
  text-align: right;
  width: 25%
}

Line 156 of the [Github](https://github.com/freeCodeCamp/freeCodeCamp/blob/28ad751427db0d622f3ef0c79daceca74137f27f/curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f46e270702a8456a664f0df.md?plain=1#L156) file within the freeCodeCamp curriculum.

Impact:

While functionally harmless in modern browsers, adding the semicolon improves code consistency and maintainability.

Discussion:

The comments on the issue highlight:
It's likely a typo since other CSS rules in the challenge have semicolons.
While not strictly an error, adding the semicolon for consistency is recommended.

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-basic-css-by-building-a-cafe-menu/step-85

### Steps to Reproduce

1. Go to '[Github](https://github.com/freeCodeCamp/freeCodeCamp/blob/28ad751427db0d622f3ef0c79daceca74137f27f/curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f46e270702a8456a664f0df.md?plain=1)'
2. Click on 'Code Edit'
3. Scroll down to 'Line 156'
4. Add ; at last character

### Expected behavior

The default code has not ; sign
In css part
It does look like a typo: https://github.com/freeCodeCamp/freeCodeCamp/blob/28ad751427db0d622f3ef0c79daceca74137f27f/curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f46e270702a8456a664f0df.md?plain=1#L156

### Screenshots

image
image

### System

- Device: Laptop
- OS: Windows 11
- Browser: edge

### Additional context

Current Status:

The issue is labeled as help wanted and type: bug.
Multiple pull requests have been linked that might fix this issue:
Update 5f3ef6e0f8c230bdd2349716.md by sharabshrestha99
fix(curriculum): typo error on css file by brunades
Additional Notes:

A comment offers a reference for further details on semicolons in CSS: https://developer.mozilla.org/en-US/docs/Web/CSS/Mozilla_Extensions

### PR DESCRIPTION
**Pull request by brunades**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #56284

Fixed all the semicolon typo errors on the css files of learn-basic-css-by-building-a-cafe-menu

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/quiz-basic-html/66df3b712c41c499e9d31e5b.md
Comment (gikf) [inline curriculum/challenges/english/25-front-end-development/quiz-basic-html/66df3b712c41c499e9d31e5b.md:1]: Please revert change in this file, it's unrelated to the linked issue.

Comment (brunades) [reply curriculum/challenges/english/25-front-end-development/quiz-basic-html/66df3b712c41c499e9d31e5b.md:1]: Thank you for reviewing my pull request. 
As per your request, I have reverted the changes to the unrelated file. The revert has been committed to the same branch, and the PR should now reflect only the intended changes.
Could you confirm if everything looks correct or if further adjustments are needed?

**Review by Sembauke — APPROVED** [review]

**Review by majestic-owl448 — APPROVED** [review]
LGTM 👍

**Comment by ilenia-magoni** [comment]
Hi @brunades. Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**